README.md

```diff
@@ -2,727 +2,369 @@
-    * ```$ mbedls --mock MID:PLATFORM_NAME``` or
-    * ```$ mbedls --mock MID1:PLATFORM_NAME1,MID2:PLATFORM_NAME2```
-    * Example: `$ mbedls --mock 0818:NUCLEO_F767ZI`
-* Remove masking with '!' prefix: `$ mbedls --mock !MID`
-* Remove all maskings using !* notation: `$ mbedls --mock !*`
-* Combine above using comma (`,`) separator: `$ mbedls --mock MID1:PLATFORM_NAME1,!MID2`
+The name of the platform that is returned for any platform that has a `target_id` that matches the first four characters specified in `mid`.
 
-## Mocking example with Freescale K64F platform
-Initial setup with 1 x Freescale ```K64F``` board:
-```
-$ mbedls
-+--------------+---------------------+------------+------------+-------------------------+
-|platform_name |platform_name_unique |mount_point |serial_port |target_id                |
-+--------------+---------------------+------------+------------+-------------------------+
-|K64F          |K64F[0]              |F:          |COM146      |02400221A0811E505D5FE3E8 |
-+--------------+---------------------+------------+------------+-------------------------+
-```
+##### `oper`
 
-* We can mask current mapping ```0240``` -> ```K64F``` to something else. For example we can replace ```K64F``` name with maybe more suitable for us in current setup ```FRDM-K64F```:
-```
-$ mbedls --mock 0240:FRDM_K64F
-```
-Current mocking mapping is stored in local file ```.mbedls-mock```:
-```
-$ cat .mbedls-mock
-{
-    "1234": "NEW_PLATFORM_1",
-    "0240": "FRDM_K64F"
-}
-```
-We can observe changes immediately. Please note this change only works in the same directory because we save ```.mbedls-mock``` file locally:
-```
-$ mbedls
-+--------------+---------------------+------------+------------+-------------------------+
-|platform_name |platform_name_unique |mount_point |serial_port |target_id                |
-+--------------+---------------------+------------+------------+-------------------------+
-|FRDM_K64F     |FRDM_K64F[0]         |F:          |COM146      |02400221A0811E505D5FE3E8 |
-+--------------+---------------------+------------+------------+-------------------------+
-```
+**Default:** `'+'`
 
-* We can remove mapping ```1234``` -> Anythying using ```!``` wild-card.
-Note: We are using flag ```-json``` to get JSON format output of the ```--mock``` operation.
-```
-$ mbedls --mock !1234 --json
-{
-    "0240": "FRDM_K64F"
-}
-```
+If set to `'+'`, the mocked platform is enabled. If `'-'`, the mocked platform is disabled.
 
-* We can add multiple mappings at the same time:
-```
-$ mbedls --mock 0000:DUMMY,1111:DUMMY_2 --json
-{
-    "1111": "DUMMY_2",
-    "0240": "FRDM_K64F",
-    "0000": "DUMMY"
-}
-```
+---
 
-* We can remove (```!```) all mappings using ```*``` wildcard:
-```
-$ mbedls --mock !*
-```
+# Testing
 
-We can verify our mapping is reset:
-```
-$ cat $HOME/.mbed-ls/.mbedls-mock
-{}
-```
+All tests are contained within the `/test` directory. You can run the tests with the following command:
```

Comment:
Note to self: Change first sentence to "The `/test` directory contains all tests." for active voice after responses to comments, so you don't lose comments.